backend/test/emr/test_create_emr_cluster.py

```diff
@@ -157,7 +157,7 @@ def _expected_args(custom_ami: Optional[str] = None):
             "Ec2KeyName": mock_cluster_config["ec2-key"],
```

Comment:
I think we're missing a test where we explicitly past in a subnet in the event body.

Reply:
ill create a new one that is more explicit